lib/livebook/intellisense.ex

```diff
@@ -383,16 +383,18 @@ defmodule Livebook.Intellisense do
 
   defp format_details_item({:module, module, _display_name, documentation}) do
     join_with_divider([
+      format_hexdoc(module),
       code(inspect(module)),
       format_documentation(documentation, :all)
     ])
   end
 
   defp format_details_item(
-         {:function, module, name, _arity, _type, _display_name, documentation, signatures, specs,
+         {:function, module, name, arity, _type, _display_name, documentation, signatures, specs,
           meta}
        ) do
     join_with_divider([
+      format_hexdoc(module, name, arity),
```

Comment:
I think we can merge this with `format_meta(:since, meta)`, wdyt? Something like:

    Hexdocs · Since v1.13.0

Reply:
In pseudo-code, we should have something like this:

```
join_with_divider([
      format_signatures(signatures, module) |> code(),
      join_with_middle_dot([
        format_hexdoc(module, name, arity),
        format_meta(:since, meta),
      ])
      format_specs(specs, name, @extended_line_length) |> code(),
      format_meta(:deprecated, meta),
      format_documentation(documentation, :all)
    ])
```

where `join_with_middle_dot` is like `join_with_divider` but it uses `" · "` for joining!

Reply:
> I think we can merge this with `format_meta(:since, meta)`, wdyt? Something like:
> 
> ```
> Hexdocs · Since v1.13.0
> ```

That sounds good!